packs/ansible/actions/vault.decrypt.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: "vault.decrypt"
+description: "Decrypt ansible data files"
+runner_type: "local-shell-script"
+entry_point: "ansible_vault.py"
+enabled: true
+parameters:
+  kwarg_op:
+    description: "Lock operator type to '--'"
+    type: string
+    default: "--"
+    immutable: true
+  sudo:
+    description: "Lock sudo, the behavior is controlled by ansible 'become_' options"
+    type: boolean
+    default: true
+    immutable: true
+  timeout:
+    description: "Action timeout in seconds. Action will get killed if it doesn't finish in timeout seconds. Lock as uneeded"
+    type: integer
+    default: 60
+    immutable: true
+  action:
+    description: "Action to use against playbook file(s)"
+    type: string
+    position: 0
+    default: decrypt
+    immutable: true
+  files:
```

Comment:
It would probably also be good to add a warning / note here that this file will be modified (encrypted) in place.

Same goes for decrypt.